libs/@guardian/core-web-vitals/package.json

```
@@ -8,13 +8,13 @@
 		"@guardian/libs": "12.0.0",
 		"tslib": "2.4.1",
 		"typescript": "4.3.2",
-		"web-vitals": "2.0.0"
+		"web-vitals": "3.1.1"
```

this should be pinned the bottom of the peerdep range

```suggestion
		"web-vitals": "3.0.0"
```

---

But I would wager that we might actually want `^3.1` in practice!

---

this allows consumers to install 3.1.0 if they want to, this package should still be compatible with it. why restrict consumers to a version of a peer dependency that we don't actually need?